api/src/integrations/core/datasource.js

```diff
@@ -547,7 +547,7 @@ class CoreDataSource {
           const updateCollection = {
             _sourceRefCoreCollectionId: amendment.permit_amendment_guid,
             name: amendment.description,
-            date: amendment.issue_date,
+            date: new Date(amendment.issue_date),
```

Comment:
This needs checking if `issue_date` isn't null, otherwise you create EPOCH dates.